kratos/python/add_properties_to_python.cpp

```diff
@@ -178,6 +179,8 @@ void  AddPropertiesToPython(pybind11::module& m)
     .def("HasTables", &Properties::HasTables)
     .def("IsEmpty", &Properties::IsEmpty)
     .def("__str__", PrintObject<Properties>)
+
+    .def("Clone", [](Properties::Pointer pSelf){ return Kratos::make_shared<Properties>(*pSelf); })
```

Comment:
Yep thats also what I had in mind 👍 

Although I am checking now the pybind docs if there is a native solution

Reply:
Seems like it should be possible to expose the copy constructor
Then the py and cpp implementation would be consistent

Reply:
But how could python know when to use the copy constructor and when to just create a "reference" - because we need to avoid that python makes copies whenever we write 
```python
property2 = property1
```
that is not pythonic...

Reply:
Or do you mean 
```python
property2 = KM.Properties(property1)
```